Sources/MapboxCoreNavigation/MBXInfo.plist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>CFBundleDevelopmentRegion</key>
+	<string>en</string>
+	<key>CFBundleExecutable</key>
+	<string>$(EXECUTABLE_NAME)</string>
+	<key>CFBundleIdentifier</key>
+	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+	<key>CFBundleInfoDictionaryVersion</key>
+	<string>6.0</string>
+	<key>CFBundleName</key>
+	<string>$(PRODUCT_NAME)</string>
```

Comment:
There are other places where we’re using `Bundle.mapboxCoreNavigation` and `Bundle.mapboxNavigationIfInstalled` to get Info.plist keys:

https://github.com/mapbox/mapbox-navigation-ios/blob/6cc9728a615ed9b08632a6c2d09f7fca57a5c542/Sources/MapboxCoreNavigation/URLSession.swift#L19

The Xcode build system normally resolves placeholder tokens in Info.plist in one of the last build phases, but it won’t process this property list the same way. As a result, the line above will literally put `$(PRODUCT_NAME)` in the user agent string instead of `MapboxCoreNavigation`, and instead of `MapboxNavigation` it’ll substitute the host application’s name.

Until SPM gains support for Apple Generic Versioning, update-version.sh will need to set `CFBundleName` to the expected name:

```bash
plutil -replace CFBundleName -string 'MapboxCoreNavigation' MBXInfo.plist
```

(likewise for Source/MapboxNavigation/MBXInfo.plist)

Reply:
Thanks for the suggestion! I'll fix this part.

Reply:
Since we haven’t released yet, we’ll need to manually run the `plutil` commands to replace these tokens.